src/lib/html.ts

```diff
@@ -21,6 +22,18 @@ export function generateHtml(
     webtreemapCss: btoa(fs.readFileSync(require.resolve('./vendor/webtreemap.css'))),
   };
 
+  // Get webtreemap data to update map on bundle select
+  let treeData = exploreResults.map<WebTreeData>((data) => ({
+    name: data.bundleName,
+    data: getWebTreeMapData(data.files),
+  }));
+
+  if (treeData.length > 1) {
+    treeData = [makeMergedTreeDataMap(cloneDeep(treeData))].concat(treeData);
+  }
+  const treeDataMap = { ...treeData };
+  const template = getFileContent(path.join(__dirname, 'tree-viz.ejs'));
```

Comment:
- Tranforms `exploreResults` to `treeData`
- Adds the combined treeData ahead
- Converts array of treeData to `treeDataMap`
- Then allows to combine `exploreResults` (see code below)